### PR TITLE
Clarify docs around using either Consul or Vault managed PKI paths

### PR DESCRIPTION
**Pull request by twunderlich-grapl**
### Description
Describe why you're making this change, in plain English.

The current docs can be misread to indicate that you need both the
Consul and Vault managed PKI Paths policies. The [Learning Tutorial](https://learn.hashicorp.com/tutorials/consul/vault-pki-consul-connect-ca?in=consul/vault-secure#create-vault-policies)
is clearer. This tries to make the docs as clear as the
learning tutorial

### Testing & Reproduction steps
* In the case of bugs, describe how to replicate
* If any manual tests were done, document the steps and the conditions to replicate
* Call out any important/ relevant unit tests, e2e tests or integration tests you have added or are adding

### Links
Include any links here that might be helpful for people reviewing your PR (Tickets, GH issues, API docs, external benchmarks, tools docs, etc). If there are none, feel free to delete this section.
 [Learning Tutorial](https://learn.hashicorp.com/tutorials/consul/vault-pki-consul-connect-ca?in=consul/vault-secure#create-vault-policies)

Please be mindful not to leak any customer or confidential information. HashiCorp employees may want to use our internal URL shortener to obfuscate links.

### PR Checklist

* [ ] updated test coverage
* [x] external facing docs updated
* [x] not a security concern
* [ ] checklist [folder](./../docs/config) consulted

### CODE REVIEW

**Comment by twunderlich-grapl** [comment]
@blake @boruszak, can you take a look at this when you have the chance?

**Review by blake — APPROVED** [review]

**Comment by david-yu** [comment]
Checks seem to be failing so tried closing/re-opening and re-running checks.

**Comment by twunderlich-grapl** [comment]
@david-yu, I've just merged main into this branch because some of the tests were failing. I'll probably need someone to authorize some of the tests

**Comment by david-yu** [comment]
Thanks it looks a bit better now that you've pulled in some other changes.
